pysid/identification/recursive.py

```diff
@@ -0,0 +1,266 @@
+# -*- coding: utf-8 -*-
+
+"""
+Module for recursive methods
+"""
+
+from numpy import zeros, identity, matmul, empty, insert, concatenate, power
+from .models import polymodel
+from .solvers import qrsolm
+
+def var_erro(error, th = 0.001):
+    """
+    
+    Measures if the error between predictions is,percentage, less than th 
+    
+    internal use
+    Parameters
+    ----------
+    error : numpy array
+    th : float
+        DESCRIPTION. The default is 0.001.
+
+    Returns
+    -------
+    bool.
+
+    """
+    #Caso o erro tenha variado menos que th entre as iterações, retorna False e sai do while
+    if error[0,0] != 0 :
+        for i in range(error.shape[1]):
+            if (abs((error[0,i] - error[1,i])/error[0,i])) > th :
+                return True
+        return False
+    else:
+        return True
+    
+def emq(na,nb,nc,nk,u,y,th,n_max):
```

Comment:
Add standard values for th and n_max:

emq(na, nb, nc, nk, u, y, th=1e-5, nmax=500)